android/src/main/java/com/rivereactnative/RiveReactNativeView.kt

```diff
@@ -271,6 +270,7 @@ class RiveReactNativeView(private val context: ThemedReactContext) : FrameLayout
   }
 
   private fun setUrlRiveResource(url: String, autoplay: Boolean = riveAnimationView.autoplay) {
+    val httpClient = HttpClient()
```

Comment:
We've got a http loader in the Android runtime; seems a shame to have this implemented twice in Kotlin.

We didn't use the ViewModel stuff as that's tied to lifecycle and you don't get lifecycle events in views (fragments and activities only). Runtime is using Volley to makes things simple and straightforward. (https://github.com/rive-app/rive-android/blob/master/kotlin/src/main/java/app/rive/runtime/kotlin/RiveAnimationView.kt#L164).

I wonder if there's a way to get RN to call down to what's happening in the Android runtime to save on duping code? Happy to help on the Android side if we need to abstract things out a bit.